src/render/pic.rs

```diff
@@ -0,0 +1,78 @@
+//! PIC rendering support.
+//!
+//! # Example
+//!
+//! ```
+//! use qrcode::QrCode;
+//! use qrcode::render::pic;
+//!
+//! let code = QrCode::new(b"Hello").unwrap();
+//! let pic = code.render::<pic::Color>().build();
+//! println!("{}", pic);
+
+#![cfg(feature = "pic")]
+
+use std::fmt::Write;
+use std::marker::PhantomData;
+
+use crate::render::{Canvas as RenderCanvas, Pixel};
+use crate::types::Color as ModuleColor;
+
+/// A PIC color.
+#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Color<'a>(pub &'a str);
+
+impl<'a> Pixel for Color<'a> {
+    type Canvas = Canvas<'a>;
+    type Image = String;
+
+    fn default_color(color: ModuleColor) -> Self {
+        Color(color.select("#000000", "#ffffff"))
+    }
+}
+
+#[doc(hidden)]
+pub struct Canvas<'a> {
+    pic: String,
+    marker: PhantomData<Color<'a>>,
+}
+
+impl<'a> RenderCanvas for Canvas<'a> {
+    type Pixel = Color<'a>;
+    type Image = String;
+
+    fn new(width: u32, height: u32, dark_pixel: Color<'a>, light_pixel: Color<'a>) -> Self {
+        Canvas {
+            pic: format!(
+                concat!(
+                    r#"command ".if !mqrcode_fg .defcolor qrcode_fg rgb {fg}""#,
+                    "\n",
+                    r#"command ".if !mqrcode_bg .defcolor qrcode_bg rgb {bg}""#,
+                    "\n",
+                    "maxpswid={w};maxpsht={h};movewid=0;moveht=1;boxwid=1;boxht=1\n",
+                    r#"define P {{ box wid $3 ht $4 color "qrcode_fg" with .nw at $1,-$2 }}"#,
+                    "\n",
+                    r#"box wid maxpswid ht maxpsht color "qrcode_bg" with .nw at 0,0"#,
```

Comment:
```suggestion
                    r#"box wid maxpswid ht maxpsht fill qrcode_bg with .nw at 0,0"#,
```

Reply:
The color names must be quoted, because AFAIK PIC does not offer arbitrary definition of variables.

Reply:
okay

actually I find that the color specification is conflicting among different dialects, could you make it

```pic
box wid maxpswid ht maxpsht {bg} with .nw at 0,0
```

and make the usage `pic::Color("color \"#ff0000\"")` or `pic::Color("fill 0xff0000")` 🤔

Reply:
While I was writing [this comment](https://github.com/kennytm/qrcode-rust/pull/65#discussion_r1661818111) you wrote the one above 🙂 

It seems you also prefer to remove support for colors. I'll go ahead and make the necessary changes.